core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -125,4 +128,18 @@ public UnderFileSystemConfiguration createMountSpecificConf(Map<String, String>
     ufsConf.mShared = mShared;
     return ufsConf;
   }
+
+  /**
+   * @param userPropertiesOnly  whether to use only the user properties
+   * @param options options for formatting the configuration values
+   * @return a map from all configuration property names to their values; values may potentially be
+   *         null
+   */
+  public Map<String, String> toMap(boolean userPropertiesOnly, ConfigurationValueOptions options) {
+    Map<String, String> map = new HashMap<>();
+    Set<PropertyKey> keys = userPropertiesOnly ? userKeySet() : keySet();
```

Comment:
What are user properties? Are UFS properties user properties? If so, this parameter `userPropertiesOnly` should always be `true`?

Reply:
I refactored the method to make it clear.